packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -2066,7 +2066,9 @@ function commitDeletionEffectsOnFiber(
           nearestMountedAncestor,
           deletedFiber,
         );
-        releaseResource(deletedFiber.memoizedState);
+        if (deletedFiber.memoizedState) {
```

Comment:
When would it not be set here?

Reply:
If the fiber is a HostResource but the props preclude getting a resource the memoized state will be null. Imagine you update a style sheet to not have a precedence or something. The old resource is released and no new resource is acquired